.env.development

```diff
@@ -0,0 +1,9 @@
+REVALIDATE_SECRET=test
+FE_HOSTNAME=localhost
+# FE_PORT is only for mock API use; only change FE_PORT if custom port is given in npm script `dev` in package.json
+FE_PORT=3000
+FE_HOST=http://$FE_HOSTNAME:$FE_PORT
```

Comment:
Variables can be used because we handle them in `lib/env.mjs` (another file in this PR).